dockerfiles/dev.Dockerfile

```diff
@@ -18,11 +18,9 @@ RUN apt-get -qq update && \
 
 COPY requirements.txt /${APP_HOME}
 
-RUN curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py \
-    && python get-pip.py \
-    && rm get-pip.py \
-    && pip install virtualenv \
-    && pip install -r requirements.txt
+RUN python -m pip install --upgrade pip \
+	&& pip install virtualenv \
+	&& pip install -r requirements.txt
```

Comment:
so, we should tweak this while we're tweaking - the thing with virtualenv is that you `pip install virtualenv` and then you create a virtualenv for your project & activate it - and only *then* do you pip install requirements. So this is currently equivalent to:

```
RUN python -m pip install --upgrade pip \
	&& pip install -r requirements.txt
```

which we could do without changing anything from how it is now. Or if we want to use a virtualenv it would be something like (ottomh)

```
RUN python -m pip install --upgrade pip \
	&& pip install virtualenv \
    && python -m virtualenv .venv \
    && source ./.venv/bin/activate
	&& pip install -r requirements.txt
```

but if we start using virtualenv we might need to change other stuff elsewhere!